wc3-hw1-TranNgocLam/cypress/e2e/registrationForm.cy.js

```diff
@@ -0,0 +1,308 @@
+          .inputUserNumber(users[4].userNumber)
+          .chooseDOB(day, month, year)
+          .uploadPicture(users[4].picture)
+          .clickSubmit()
+
+        confirmForm
+          .isCorrectName(users[4].firstName, users[4].lastName)
+          .isCorrectGender(users[4].gender)
+          .isCorrectEmail(users[4].email)
+          .isCorrectMobile(users[4].userNumber)
+          .isCorrectDOB(day, month, year)
+          .isCorrectPicture(users[4].picture)
+          .clickClose()
+
+      });
+    });
+
+    /*  TCRegister#8: Verify that user cannot submit with invalid data:
+      - Email lacks of email domain.
+      - Phone number includes characters. */
+    it.only('TCRegister#8', () => {
+      cy.get('@registration').then((users) => {
+
+        registerForm
+          .inputLastName(users[5].lastName)
+          .inputFirstName(users[5].firstName)
+          .inputEmail(users[5].email)
+          .checkGender(users[5].gender)
+          .inputUserNumber(users[5].userNumber)
+          .clickSubmit()
+
+        registerForm
+          .checkFirstNameValid(true)
+          .checkLastNameValid(true)
+          .checkEmailValid(false)
+          .isGenderChecked(true)
+          .checkMobileValid(false)
+
+      });
+    });
+
+    /*  TCRegister#9: Verify that user cannot submit with invalid data:
+      - Email's domain lacks of ".com".
+      - Phone numbers begin with a phone code instead of 0. */
+    it.only('TCRegister#9', () => {
+      cy.get('@registration').then((users) => {
+
+        registerForm
+          .inputLastName(users[6].lastName)
+          .inputFirstName(users[6].firstName)
+          .inputEmail(users[6].email)
+          .checkGender(users[6].gender)
+          .inputUserNumber(users[6].userNumber)
+          .clickSubmit()
+
+        registerForm
+          .checkFirstNameValid(true)
+          .checkLastNameValid(true)
+          .checkEmailValid(false)
+          .isGenderChecked(true)
+          .checkMobileValid(false)
+
+      });
+    });
+  /*    TCRegister#10: Verify that user cannot submit with invalid data:
+      - Email's domain lacks of ".com".
+      - Phone numbers begin with a phone code instead of 0. */
+    it.only('TCRegister#10', () => {
+      cy.get('@registration').then((users) => {
+
+        registerForm
+          .inputLastName(users[7].lastName)
+          .inputFirstName(users[7].firstName)
+          .inputEmail(users[7].email)
+          .checkGender(users[7].gender)
+          .inputUserNumber(users[7].userNumber)
+          .clickSubmit()
+
+        confirmForm
+          .isCorrectName(users[7].firstName, users[7].lastName)
+          .isCorrectGender(users[7].gender)
+          .isCorrectEmail(users[7].email)
+          .isCorrectMobile(users[7].userNumber)
+          .clickClose()
+
+      });
+    });
+
+  /*    TCRegister#11: Verify that user cannot submit with already registered mobile phone and email. */
+  it.only('TCRegister#11', () => {
```

Comment:
As I said before, we assume that this website are stable without bug, so this case won't make sense.
But for the idea, this case is also wrong. Your case are cannot submit with same email and mobile but below you verify the submit successfully form

Reply:
In this case, when I manually ran the above case, it turned out that I could submit with the same email and mobile which means this case failed. So, I came up with the idea that if it submits successfully, it fails. That's the reason why I checked whether the confirmation form will pop up or not.

Reply:
As I understand you are assuming that can submit with same email and mobile is a bug? If so, the expected result is not the popup is shown, it can be email and mobile border is red or something like that. You assume this is a bug but your script status is bug, it won't make sense, you have to write your script base on your expected result